modules/core/src/utils/typed-array-manager.js

```diff
@@ -0,0 +1,66 @@
+export class TypedArrayManager {
+  constructor({overAlloc = 1, poolSize = 5} = {}) {
```

Comment:
5 seems like a very small limit to me. I'm thinking let it grow to the size the application needs?